### PR TITLE
Add channelCount to audio track settings, constraints, capabilities

### PR DESCRIPTION
**Pull request by joeberkovitz**
LC proposal documented in https://lists.w3.org/Archives/Public/public-media-capture/2015Jun/0031.html to address lack of channel count information in audio track capabilities.

### CODE REVIEW

**Comment by stefhak** [comment]
This LGTM.

**Comment by burnburn** [comment]
I haven't seen any list support for this yet.  Is there anyone who can give some insight as to who might implement this?

**Comment by alvestrand** [comment]
Mozilla has said on the list that they will implement this.
Google has people arguing for this who are part of the WebAudio team.

File: getusermedia.html [hunk 1]
```diff
@@ -1261,6 +1265,10 @@
 
           <dd />
 
+          <dt>(long or LongRange) channelCount</dt>
```
Comment (jan-ivar) [inline getusermedia.html]: Are we sure we want a range and not an array of numbers?

Comment (alvestrand) [inline getusermedia.html]: that's a list-discussion item. The Audio WG had earlier abandoned the idea of a list of configurations (2, 5.1, 7.2), and gone for a number instead. If it's a number, not an identifier, I think ranges are the correct treatment.

Comment (joeberkovitz) [inline getusermedia.html]: I would say that this should be treated the same as `sampleRate`, which is currently a range (even though it is likely that only discrete values within that range are acceptable values, much as for `channelCount`).

Comment (jan-ivar) [inline getusermedia.html]: OK just checking, no need for discussion on my account.

**Comment by burnburn** [comment]
This LGTM.

**Pull request merged**